### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by marzpuente**
support files for flask camera server

### CODE REVIEW

**Comment by benhylak** [comment]
Can you add a screenshot of your work, any current limitations, etc.

**Comment by benhylak** [comment]
@marzpuente bump

**Comment by marzpuente** [comment]
@benhylak here are some screenshots. Picamera works great. I just need to run Mona's thermal code to pipe the images to the program that reading the images and animates them on the page.
sprintfeb102
sprintfeb10

**Review by benhylak — COMMENTED** [review]
remove extra image files (1, 2, 3.jpg)

File: SW/Python/flask-video-streaming-master/app.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
 if os.environ.get('CAMERA'):
     Camera = import_module('camera_' + os.environ['CAMERA']).Camera
 else:
-    from camera import Camera
+    from camera_pi import Camera
+    from camera import CameraThermal
```
Comment (benhylak) [inline SW/Python/flask-video-streaming-master/app.py:11]: rename to ThermalCamera

File: SW/Python/flask-video-streaming-master/base_camera.py.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
+class BaseCamera(object):
```
Comment (benhylak) [inline SW/Python/flask-video-streaming-master/base_camera.py.txt:1]: delete this file? (it's a .txt?)

**Review by benhylak — CHANGES_REQUESTED** [review]
To be honest, I'm super confused.

You have a camera.py file, a base_camera.py file, base_thermal_camera.py file, a camera_pi.py, among others. The base_camera_thermal.py. 

The camera.py file, which has the ThermalCamera class, is just reading in images instead of actually reading in from the thermal camera (which is what we need...)

base_camera_thermal and base_camera seem to be the exact same thing, more or less. Not sure what the difference is.

.pyc files should not be committed to the repo. They're just artifacts of the build process.

Classes and files should be labeled or self explanatory. 

Also, please be clearer about what does and does not work. The thermal camera class appears to work but is just looping through 4 images. That can lead to a ton of confusion down the line.

**Pull request closed without merging**